app/_src/gateway/production/deployment-topologies/hybrid-mode/hybrid-mode-spec.md

```diff
@@ -0,0 +1,114 @@
+
+If the CP determines that the DP is using a compatible version of {{site.base_gateway}}, it will immediately export the current configuration and push it to the DP. This ensures that newly connected DPs always receive the latest configs on start-up. If the DP's version is not compatible with the CP's version a sync will not be allowed. 
+
+
+## Configuration management
+
+When a Kong admin makes a change to the Kong database entity, hybrid mode notices such change and builds a new config. The config is then pushed to all DPs that are currently connected in a WebSocket frame with the following format: 
+
+```
+{ "type": "reconfigure",
+  "config_table": {
+    "_format_version": "1.1",
+    "consumers": [
+      {
+        "username": "user",
+        "tags": [
+          "internal-user"
+        ],
+        "plugins": [
+          {
+            "name": "rate-limiting",
+            "enabled": true,
+            "protocols": [
+              "http",
+              "https"
+            ]
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+
+All frames transferred from the CP to the DP are Gziped before being sent over WebSocket. This reduces the size of the payload significantly since Kong configs are highly compressible.
+
+### Sensitive data management 
+
+Because the CP node dumps the entire database config and send it to the DP, the following data may be included: 
+
+* Private key of TLS certificate for proxied domains.
+* IP address, port, client certificates, private keys and any other information used to access upstream services.
+* Consumer information including emails, authentication plugin credentials like an API Key for the key-auth plugin.
+* Unique ID for identifying the hybrid mode cluster. This is generated once and shared by every CP and DP within the same cluster. This ID is used for anonymous reporting and if anonymous reporting is not disabled.
+
+The DP nodes also store the latest received configuration data on the filesystem in order to survive CP failures or streamline DP startup.
+
+## Heartbeat and version reporting
+
+The CP needs to know what the config version of each DP that is running in order to know if the DP is up to date, or if a config has failed. The config version is reported in the WebSocket Ping frame that the DP generates every 30 seconds. If a new config was pushed and applied successfully, then a Ping frame is generated immediately by the DP in order to report the version change to the CP. The Ping frame is also used to keep the TCP connection open so that configs can be pushed immediately. 
+
+
+## FAQ
+
+### How long can a disconnected DP continue working without impact to operations? 
+
+Indefinitely. Because DP nodes cache the last known config from the CP, they will continue to function with that config indefinitely, even after restarts. However, there are exceptions that are worth noting. 
+
+1. If the EE license expires the DP may stop working. 
```

Comment:
In the case of license expired, I think the data plane will continue to run but the enterprise plugins will not work.